### PR TITLE
feat: Eval meta

### PR DESCRIPTION
**Pull request by Ciroye**

### CODE REVIEW

File: backend/app/domain/services/builder_and_evaluation/evaluation.py [hunk 1]
```diff
@@ -414,8 +413,8 @@ def evaluation(
         model_s3_zip: str,
         model_id: int,
         user_id: int,
-        evaluate_no_scoring_datasets: bool = False,
         selected_langs: str = None,
+        evaluate_no_scoring_datasets: bool = False,
```
Comment (remg1997) [inline backend/app/domain/services/builder_and_evaluation/evaluation.py:417]:
```suggestion
        evaluate_not_scoring_datasets: bool = False,
```

File: frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              setChatHistory={setChatHistory}
+              updateModelInputs={updateModelInputs}
+              setIsGenerativeContext={setIsGenerativeContext}
+            />
+          </div>
+          <div className="col-span-1">
+            <div className="p-4 bg-white">
+              <h2 className="mb-4 text-2xl font-bold">Instructions</h2>
+              <p className="text-gray-600">
+                You are playing the part of a 47-year-old female patient who is
```
Comment (remg1997) [inline frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx:73]: Any reason you want to keep this placeholder?

File: frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  const [provider, setProvider] = useState("");
+  const [artifactsInput, setArtifactsInput] = useState<any>(
+    generative_context.artifacts,
+  );
+  const [finishConversation, setFinishConversation] = useState(false);
+  const { updateModelInputs } = useContext(CreateInterfaceContext);
+  const { user } = useContext(UserContext);
+
+  const checkIfUserIsSignedInConsent = async () => {
+    const signConsent = await post("/task/check_sign_consent", {
```
Comment (remg1997) [inline frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx:31]: We should rename the endpoint we're calling to check_signed_consent. Not urgent, but definitely an improvement.

**Review by remg1997 — APPROVED** [review]
Left a couple of comments. I don't think any are required changes though.

File: frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx (hunk 2 above)
Comment (Ciroye) [reply frontends/web/src/new_front/components/CreateSamples/CreateSamples/AnnotationInterfaces/Contexts/ChatWithInstructions.tsx:73]: To be replaced in the next PR

**Pull request merged**